interface/globals.php

```diff
@@ -124,6 +124,7 @@ function undoMagicQuotes($array, $topLevel=true) {
 if (!$disable_utf8_flag) {    
  ini_set('default_charset', 'utf-8');
  $HTML_CHARSET = "UTF-8";
+ mb_internal_encoding('UTF-8');
 }
 else {
  ini_set('default_charset', 'iso-8859-1');
```

Comment:
would it make sense to add the following here? :
``````
mb_internal_encoding('ISO-8859-1');